src/edu/pugetsound/mathcs/nlp/architecture_nlp/features/FoldCreator.java

```diff
@@ -0,0 +1,211 @@
+package edu.pugetsound.mathcs.nlp.architecture_nlp.features;
+import java.io.*;
+
+import java.util.Random;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Scanner;
+
+import com.sun.xml.internal.bind.v2.runtime.unmarshaller.XsiNilLoader.Array;
+
+import edu.pugetsound.mathcs.nlp.lang.Conversation;
+import edu.pugetsound.mathcs.nlp.lang.Utterance;
+import edu.pugetsound.mathcs.nlp.kb.KBController;
+import edu.pugetsound.mathcs.nlp.architecture_nlp.features.TextAnalyzer;
+
+/**
+ * 
+ * Contains a main method which takes 2 files, one full of words and their parts of speech, and another with 
+ * sentences and their logical parses. Together it creates a new file with new sentences from the word file paired
+ * with the logical parses from the other. 
+ * REQUIRES A words.txt and test.ccg files to operate. 
+ *  
+ * @author NathanielLivingston
+ */
+// pull all code put in spf read in geo file push out fold push request keep verb the same!
+
+
+public class FoldCreator{ 
+
+	/**
+	 * @param args
+	 */
+	public static void main(String [] args)
+	{
+		System.out.println(System.getProperty("user.dir"));
+		try {
+			
+			String line = null;
+			
+			FileReader fileReader = new FileReader("src/edu/pugetsound/mathcs/nlp/architecture_nlp/features/words.txt"); // open the word file
```

Comment:
Any input files should be in a different directory. Either:
1. resources/switchboard_scrubbed3
2. resources/SpfResources
Also, look at the PathFormat class in the /src/edu/pugetsound/mathcs/nlp/util directory. This class has a method that takes a relative path (i.e. your path that starts with src) and converts it into an absolute path based upon your system. Use this instead of passing in "src/edu/..." straight into your FileReader class